### ISSUE TITLE
Problem with snapshots when compiling

### ISSUE THREAD

**Issue by Flobbes** [tinyMediaManager/tinyMediaManager#30]
When I try to compile the tinymediamanager devel branch, eclipse complains:

Description	Resource	Path	Location	Type
Missing artifact org.tinymediamanager:api-scraper:jar:2.0-SNAPSHOT	pom.xml	/tinyMediaManager	line 28	Maven Dependency Problem

It does it more or less for every snapshot.

I think I added https://oss.sonatype.org/content/repositories/snapshots/, like stated in the readme, correctly, but I'm not completly sure.
Am I doing something wrong?

**Comment by myron0815** [comment]
when the repository is added correctly in ~/.m2/settigs.xml, it should work (although maven caches quite often)
please try to build once with "mvn package -U"

Alternatively, checkout the https://github.com/tinyMediaManager/api-scraper project inside your IDE...

**Comment by Flobbes** [comment]
Ok got it.
I made a mistake with the settings.xml.
The file did not exist for me, so I created an empty own and did not know that I would need some surrounding tags.

Furthermore I had to remove the last classpathentry lines in .classpath refurring to the other projects.
Otherwise I always got error messages that needed projects were not available.
Now compilation works.

Thank you very much for the help.
